base_substate/tests/test_base_substate.py

```diff
@@ -106,12 +115,10 @@ def test_sale_order_substate(self):
         )
         self.assertTrue(so_test1.state == "draft")
         self.assertTrue(so_test1.substate_id == self.substate_under_negotiation)
-
-        # Test that validation of sale order change substate_id
+        # Test that validation of sale order change substate_id and send email
```

Comment:
Thanks for all your work on this. ❤️  Can you actually test the mail message?

Reply:
OK, I added test mail message in test script please check.